src/main/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import nEws
+
+class News(admin.ModelAdmin):
+    list_display=('order_no','content','do_show','document')
+
+admin.site.register(nEws,News)
```

Comment:
Look the rest of my code to look at the updated syntax of `registering on the admin